src/legacy/core_plugins/kibana/public/discover/np_ready/angular/discover.js

```diff
@@ -675,17 +675,6 @@ function discoverController(
         }
       });
 
-      $scope.$watch('vis.aggs', function() {
```

Comment:
The watcher didn't work due to the API changes, decided it's even not needed and removed it in favor of a simpler solution